README.md

```diff
@@ -622,6 +625,23 @@ Also the device info as follows:
 
 Depending upon the config chosen to build, `DEVICE_REVISION` will be `Rev4 with CC2520` or `Rev5 with CA8210`.
 
+
+###MAC Address Scheme
+
+On bootup, mac address init script will check for mac address in OTP, else it will check for /etc/config/MAC_x,
+else it will generate MAC from flash serial number stored in OTP, else it will generate the mac addres randomly
+for each interface and store it at /etc/config/MAC_x for the subsequent usage.
+
```

Comment:
Too many "else".  This can be put in a numbered list of top-to-bottom priority, something like:

On bootup, board will check for MAC address in the following order of priority:
1. Check from OTP
2. Check in /etc/config/MAC_x
3. Generate from flash serial number stored in OTP
4. Generate randomly for each interface and store at /etc/config/MAC_x for the subsequent usage